### ISSUE TITLE
Failures with circular dependency

### ISSUE THREAD

**Issue by awk** [sapegin/grunt-bower-concat#38]
grunt-bower-concat fails if it detects a circular dependency between two modules. Bower has supported circular dependencies for a few releases and some of the Polymer core components have circular dependencies.

Even though I'm not using grunt-bower-concat with those Polymer core components the error : Circular reference detected is thrown from resolveDependencyGraph.

As a workaround is there a way to exclude certain modules from the dependency checks? I only include: a small number of modules - and not the polymer ones.

**Comment by rurri** [comment]
Yeah this would be a good feature for us as well. Since this seems to block this from being used with Polymer.
